### PR TITLE
added conv1D example

### PR DESCRIPTION
**Pull request by mikekestemont**
Added a toy example of a model using a 1D conv layer on top of embeddings. The performance of this model seems comparable to that of the LSTM in imdb_lstm.py, reaching an acceptable 83.16% on the test data at epoch 10. All feedback welcome. I also want to add a character level 1D example but ran into the issue described in #388.

### CODE REVIEW

File: examples/imdb_conv.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+from keras.models import Sequential
+from keras.layers.core import Dense, Dropout, Activation, Flatten
+from keras.layers.embeddings import Embedding
+from keras.layers.convolutional import Convolution1D, MaxPooling1D
+from keras.datasets import imdb
+
+# set parameters:
+max_features = 5000
+maxlen = 100
+batch_size = 10
```
Comment (fchollet) [inline examples/imdb_conv.py]: With such a small batch size you are not utilizing the GPU. I have 95s/epoch with a batch size of 32 on GPU.

File: examples/imdb_conv.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+# We add a vanilla hidden layer:
+model.add(Dense(output_size, hidden_dims))
+model.add(Dropout(0.5))
+model.add(Activation('relu'))
+
+# We project onto a single unit output layer, and squash it with a sigmoid:
+model.add(Dense(hidden_dims, 1))
+model.add(Activation('sigmoid'))
+
+# we use some mild clipping to protect our gradients from vanishing:
```
Comment (fchollet) [inline examples/imdb_conv.py]: Surely you mean "exploding", not "vanishing". But exploding gradients don't occur in feed-forward networks.
